llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/base.py

```diff
@@ -0,0 +1,183 @@
+"""Tools for interacting with an Apache Cassandra database."""
+from typing import List
+
+from llama_index.core.readers.base import BaseReader
+from llama_index.core.schema import Document
+from llama_index.core.tools.tool_spec.base import BaseToolSpec
+
+
+from pydantic import Field
+
+from llama_index.tools.cassandra.cassandra_database_wrapper import (
+    CassandraDatabase,
+)
+
+
+class CassandraDatabaseToolSpec(BaseToolSpec, BaseReader):
+    """Base tool for interacting with an Apache Cassandra database."""
+
+    db: CassandraDatabase = Field(exclude=True)
+
+    spec_functions = [
+        # "cassandra_db_query",
```

Comment:
Uncomment or remove ?

Reply:
That shouldn't have been commented. Fixed